### ISSUE TITLE
“nsc generate config --nats-resolver” will output extra commas in resolver_preload

### ISSUE THREAD

**Issue by sinbane** [nats-io/nsc#650]
### What version were you using?

2.8.6

### What environment was the server running in?

k8s

### Is this defect reproducible?

yes, just copy direct output of "nsc generate config --nats-resolver"

### Given the capability you are leveraging, describe your expectation?

the output of command "nsc generate config --nats-resolver" will be something like below, there's a commas at the end of the account jwt. if someone copies this and use it in config file, will fail to start server. The correct value has no commas. 
"
# Later changes to the system account take precedence over the system account jwt listed here.
resolver_preload: {
	AD4******6U: eyJ0*********JnCQ,
}
"

### Given the expectation, what is the defect you are observing?

server fails to start since the incorrect commas

**Comment by sinbane** [comment]
https://github.com/nats-io/nsc/blob/01db6f2416be6076c564865dc29336e095af6667/cmd/natsresolverconfigbuilder.go#L87

**Comment by aricart** [comment]
Wondering how you are importing the file

**Comment by aricart** [comment]
So did some more experiment:

This is a config file that imports `auth.conf` in the same dir.
```
cat server.conf
include ./auth.conf

port: 4333
```

Save the resolver as auth.conf
```
nsc generate config --nats-resolver --config-file /tmp/auth.conf -F
[ OK ] wrote server configuration to `/tmp/auth.conf`
Success!! - generated `/tmp/auth.conf`
```

```
nats-server -c server.conf
[17415] 2024/05/21 09:51:03.628661 [INF] Starting nats-server
[17415] 2024/05/21 09:51:03.628803 [INF]   Version:  2.10.14
[17415] 2024/05/21 09:51:03.628806 [INF]   Git:      [not set]
[17415] 2024/05/21 09:51:03.628808 [INF]   Name:     NBJ5IMX4IA7M6EHIP4E4OKY2HUHCJIZT2YPQ5VGJAQCQSATMLJMNNQ2M
[17415] 2024/05/21 09:51:03.628811 [INF]   ID:       NBJ5IMX4IA7M6EHIP4E4OKY2HUHCJIZT2YPQ5VGJAQCQSATMLJMNNQ2M
[17415] 2024/05/21 09:51:03.628833 [INF] Using configuration file: server.conf
[17415] 2024/05/21 09:51:03.628837 [INF] Trusted Operators
[17415] 2024/05/21 09:51:03.628839 [INF]   System  : ""
[17415] 2024/05/21 09:51:03.628842 [INF]   Operator: "O"
[17415] 2024/05/21 09:51:03.628844 [INF]   Issued  : 2024-05-21 09:43:43 -0500 CDT
[17415] 2024/05/21 09:51:03.628866 [INF]   Expires : Never
[17415] 2024/05/21 09:51:03.629305 [INF] Managing all jwt in exclusive directory /tmp/jwt
[17415] 2024/05/21 09:51:03.629561 [INF] Listening for client connections on 0.0.0.0:4333
[17415] 2024/05/21 09:51:03.629833 [INF] Server is ready
```

**Comment by aricart** [comment]
In the above cases the contents for the auth is saved directly to a file, and it is loaded by the server without errors wondering what your setup is doing, as I have the commas in there and it works.

**Comment by aricart** [comment]
Also what server version are you using?

**Comment by sinbane** [comment]
Hi aricart, I use helm chart to install on k8s, so I copy the cmd output and use it in value.yaml, then when helm to merge the values, I think it will take the commas as a part of string value to merge. then will meet error when the instance is starting.

**Comment by aricart** [comment]
looks like you are assuming that it is yaml, but it is conf, not yaml.

**Comment by sinbane** [comment]
yes, right, because the doc said nats config is combined benifits of yaml and json, so it will let me think the output config is usable in yaml by mistake. and in the doc of helm chart, there's an example to enable resolver with helm merge, in that example says to copy the output of nsc, so I copied with comma, I thought the comma is some magic char by mistake.

so in my opinion, the comma here is a little useless, but let people confusing.
